### ISSUE TITLE
Recall is always calculated as 1.

### ISSUE THREAD

**Issue by jspark1213** [Nanne/pytorch-NetVlad#74]
Hi!
I implemented the custom dataset on this repository.
As I know, the recall is calculated using positive distance I set.
Since I didn't know what appropriate threshold for my dataset is, I set the threshold intuitively.
I increased the threshold by 1 from 1 to 20.
But the recall@N is always 1(the perfect score).
I don't know why this problem occured.
The range of utm from my dataset is lon(-164 to 378), lat(-101 to 136).
Does this problem occur because of the wrong threshold?
I swear there is no duplicated image between train set and test set.
BTW, the loss is always under the 0.2. Is it wrong?
p.s. The number of clusters is 64, default one.

And also I changed the n_jobs(argument of knn) from -1 to 1 because of the warning message to change that value.

**Comment by Nanne** [comment]
It's a bit out of scope for the issues of this repo to aid with custom datasets etc. Maybe double check whether your coordinates are correctly in UTM might be a direction to go. The distance thresholds are in meters and were originally set based on frequency of images taken by the google car.

Hope this gives you some direction, good luck!
